conda.recipe/meta.yaml

```diff
@@ -5,16 +5,23 @@ package:
 source:
   path: ../
 
+build:
+  number: {{ environ.get('GIT_DESCRIBE_NUMBER', 0) }}
+  {% if environ.get('GIT_DESCRIBE_NUMBER', '0') == '0' %}string: py{{ environ.get('PY_VER').replace('.', '') }}_0
+  {% else %}string: py{{ environ.get('PY_VER').replace('.', '') }}_{{ environ.get('GIT_BUILD_STR', 'GIT_STUB') }}{% endif %}
+
```

Comment:
OK, I found this: https://github.com/jupyter/notebook/blob/master/notebook/base/zmqhandlers.py#L90, are you taking about this???